src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -1017,19 +1027,6 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_ep_t, const uct_ep_params_t *params)
         uct_rc_iface_add_qp(&iface->super, &self->super, self->tm_qp.qp_num);
     }
 
-    addr = (uct_rc_mlx5_iface_flush_addr_t*)UCT_EP_PARAM_VALUE(params,
-                                                               iface_addr,
-                                                               IFACE_ADDR,
-                                                               NULL);
-    /* if iface address is not provided or address doesn't contain
-     * FLUSH_RKEY flag - mark flush_rkey as invalid */
-    if ((addr != NULL) &&
-        (addr->super.flags & UCT_RC_MLX5_IFACE_ADDR_FLAG_FLUSH_RKEY)) {
-        self->super.flush_rkey = (uint32_t)addr->flush_rkey_hi << 16;
-    } else {
-        self->super.flush_rkey = UCT_IB_MD_INVALID_FLUSH_RKEY;
-    }
```

Comment:
why removed? seems needed for connect to iface transports

Reply:
not sure I got question... this is rc_mlx5 transport - connect to EP supported only, rkey is updated in connect_to_ep_v2 routine

Reply:
oh, you are right